WTWSMS/decisions/realm_decisions.txt

```diff
@@ -1343,6 +1343,18 @@ decisions = {
 				factor = 0
 				religion = marcionism
 			}
+			modifier = {
+				factor = 10
+				trait = kind
+			}
```

Comment:
Just because I agree with the vanilla design it doesn't mean I'm revising history.